test/Test/Database/LSMTree/Internal/RawPage.hs

```diff
@@ -187,6 +187,8 @@ tests = testGroup "Database.LSMTree.Internal.RawPage"
     , testProperty "entry" prop_single_entry
     , testProperty "rawPageOverflowPages" prop_rawPageOverflowPages
     , testProperty "from/to reference impl" prop_fromToReferenceImpl
+    , testProperty "∀ key page. maybe True (key <=) (getRawPageIndexKey . rawPageIndex page .  id  =<< rawPageFindKey page key)" prop_findKey_index_law_GT
+    , testProperty "∀ key page. maybe True (key > ) (getRawPageIndexKey . rawPageIndex page . pred =<< rawPageFindKey page key)" prop_findKey_index_law_LT
```

Comment:
One argument in favour of simpler test names, e.g. just `prop_findKey_index_law_LT`, is that it's easier to search the code for these names, and it's easier to run the tests in isolation using `-p`. What are your thoughts?

I often just put the name of the function in the tests, and if I want to describe what a test does more succinctly then I'll put it as a haddock comment on the function

Reply:
In my own projects, for type-class laws, I have included the very verbose name of the law. However, I think that since this project has consistently used shorter test names I should modify these test names to follow suit. 

I think I'd like to, if you don't object, move the verbose "relational law" into the counter example message which appears only if the property fails to hold.